server/belga/io/feed_parsers/belga_ansa.py

```diff
@@ -115,13 +117,15 @@ def meta_parse(self, xml, item):
         item["source"] = (source_elem.text).strip()
 
         # headline
-        item["headline"] = self.get_headline(xml).strip()
+        headline_text = self.get_headline(xml).strip()
+        if headline_text.startswith(">>>ANSA/"):
+            item["headline"] = headline_text[len(">>>ANSA/") :]
```

Comment:
would be good to have a function for this to avoid duplication

Reply:
I'll probably go with it since it seems like a nice idea.